packages/devtools_server/lib/src/handlers.dart

```diff
@@ -0,0 +1,43 @@
+import 'dart:async';
+import 'dart:isolate';
+
+import 'package:path/path.dart' as path;
+import 'package:shelf/shelf.dart' as shelf;
+import 'package:shelf/shelf_io.dart' as shelf;
+import 'package:shelf_static/shelf_static.dart';
+
+/// Default [shelf.Handler] for serving DevTools files.
+///
+/// This serves files out from the build results of running a pub build of the
+/// DevTools project.
+Future<shelf.Handler> defaultHandler() async {
+  final Uri resourceUri = await Isolate.resolvePackageUri(
```

Comment:
[nit] I forget where we landed with respect to this package but I think you can omit the types here and elsewhere.

Reply:
feel free to strip all types that don't add value.

Reply:
Done.